raiden/ui/cli.py

```diff
@@ -33,13 +34,25 @@
 gevent.monkey.patch_all()
 
 
+class AddressType(click.ParamType):
+    name = 'address'
+
+    def convert(self, value, param, ctx):
+        try:
+            return address_decoder(value)
+        except BadRequestError as err:
+            self.fail('{}: {}'.format(value, err.message))
```

Comment:
Using the exception message as the user-facing warning will be confusing since it will most likely be "Addresses must be 20 or 0 bytes long" which isn't true in hex encoded form (and the "0 bytes long" part isn't helpful to end-users).

I'd simply go for a generic "Please specify a valid hex encoded address" or similar.